FiveCalls/FiveCalls/BlueButton.swift

```diff
@@ -59,10 +59,25 @@ class BlueButton: UIButton {
         }
     }
 
+    override var isEnabled: Bool {
+        didSet {
+            updateColors()
+        }
+    }
+    
     func updateColors() {
         if isHighlighted { backgroundColor = highlightBackgroundColor }
         else if isSelected { backgroundColor = selectedBackgroundColor }
         else { backgroundColor = normalBackgroundColor }
+        
+        if isEnabled {
```

Comment:
Can we maybe re-factor this to handle the `titleLabel?.alpha` rules from `isHighlighted` and `isSelected` as well?